src/main/scala/tutorial/CreateNDWIPng.scala

```diff
@@ -0,0 +1,41 @@
+package tutorial
+
+import geotrellis.raster._
+import geotrellis.raster.io.geotiff._
+import geotrellis.raster.render._
+import com.typesafe.config.ConfigFactory
+
+
+/**
+  * Created by FroehlingGallier on 10/13/16.
+  */
+object CreateNDWIPng {
+
+  val maskedPath = "data/r-g-nir.tif"
+  val ndviPath = "data/ndwi.png"
+
+  def main(args: Array[String]): Unit = {
+    val ndwi = {
+      // Convert the tile to type double values,
+      // because we will be performing an operation that
+      // produces floating point values.
+      println("Reading in multiband image...")
+      val tile = MultibandGeoTiff(maskedPath).convert(DoubleConstantNoDataCellType)
+
+      // Use the combineDouble method to map over the red and infrared values
+      // and perform the NDVI calculation.
+      println("Performing NDWI calculation...")
+      tile.combineDouble(0, 1) { (g: Double, ir: Double) =>
```

Comment:
The `0, 1` represents the band indexes. So here should change too - before band 0 was red and band 1 was nir.